lib/portlayer/event/collector/vsphere/vm_event.go

```diff
@@ -39,16 +39,12 @@ func NewVMEvent(be types.BaseEvent) *VMEvent {
 		ee = events.ContainerRemoved
 	case *types.VmGuestShutdownEvent:
 		ee = events.ContainerShutdown
-	case *types.VmRegisteredEvent:
-		ee = events.ContainerRegistered
 	case *types.VmMigratedEvent:
 		ee = events.ContainerMigrated
 	case *types.DrsVmMigratedEvent:
 		ee = events.ContainerMigratedByDrs
 	case *types.VmRelocatedEvent:
 		ee = events.ContainerRelocated
-	default:
-		panic("Unknown event")
```

Comment:
Do not remove default clause. We either need to log a missed event or explicitly assign empty string here.